static/app/utils/useMedia.tsx

```diff
@@ -29,5 +25,9 @@ export default function useMedia(query: string) {
     };
   }, [query]);
 
+  if (!window.matchMedia) {
+    return false;
+  }
```

Comment:
I wonder if it would be better to define it as something like this instead

```tsx
const useMedia = window.matchMedia
  ? useMediaHook
  : (query: string) => false;
```

Reply:
That could still cause conditional hook calls though, I wouldnt try and adopt that practice

Reply:
Oh I was suggesting this happen at the module level, at which point it would only ever export once and never change, so the hook is no longer conditional :)

Reply:
Ah, ok. Yeah, if window.matchMedia is just a feature check we def could. I think we could possibly even just drop the window.matchMedia check, seems like it has very wide support